openmc/mgxs/mgxs.py

```diff
@@ -2829,6 +2832,294 @@ def nu(self, nu):
+        compute the cross section
+    scores : list of str
+        The scores in each tally used to compute the multi-group cross section
+    filters : list of openmc.Filter
+        The filters in each tally used to compute the multi-group cross section
+    tally_keys : list of str
+        The keys into the tallies dictionary for each tally used to compute
+        the multi-group cross section
+    estimator : 'analog'
+        The tally estimator used to compute the multi-group cross section
+    tallies : collections.OrderedDict
+        OpenMC tallies needed to compute the multi-group cross section. The keys
+        are strings listed in the :attr:`TransportXS.tally_keys` property and
+        values are instances of :class:`openmc.Tally`.
+    rxn_rate_tally : openmc.Tally
+        Derived tally for the reaction rate tally used in the numerator to
+        compute the multi-group cross section. This attribute is None
+        unless the multi-group cross section has been computed.
+    xs_tally : openmc.Tally
+        Derived tally for the multi-group cross section. This attribute
+        is None unless the multi-group cross section has been computed.
+    num_subdomains : int
+        The number of subdomains is unity for 'material', 'cell' and 'universe'
+        domain types. This is equal to the number of cell instances
+        for 'distribcell' domain types (it is equal to unity prior to loading
+        tally data from a statepoint file).
+    num_nuclides : int
+        The number of nuclides for which the multi-group cross section is
+        being tracked. This is unity if the by_nuclide attribute is False.
+    nuclides : Iterable of str or 'sum'
+        The optional user-specified nuclides for which to compute cross
+        sections (e.g., 'U238', 'O16'). If by_nuclide is True but nuclides
+        are not specified by the user, all nuclides in the spatial domain
+        are included. This attribute is 'sum' if by_nuclide is false.
+    sparse : bool
+        Whether or not the MGXS' tallies use SciPy's LIL sparse matrix format
+        for compressed data storage
+    loaded_sp : bool
+        Whether or not a statepoint file has been loaded with tally data
+    derived : bool
+        Whether or not the MGXS is merged from one or more other MGXS
+    hdf5_key : str
+        The key used to index multi-group cross sections in an HDF5 data store
+
+    """
+
+    def __init__(self, domain=None, domain_type=None, groups=None, nu=False,
+                 by_nuclide=False, name='', num_polar=1, num_azimuthal=1):
+        super(DiffusionCoefficient, self).__init__(domain, domain_type, groups,
+                                                   nu, by_nuclide, name,
+                                                   num_polar, num_azimuthal)
+        self._rxn_type = 'diffusion-coefficient'
+        cv.check_value('nu for diffusion-coefficient', nu, [False])
```

Comment:
Why is `True` not allowed as a possible value for `nu`? 

Seems like it would be necessary if the downstream applications used the scattering production matrix and a reduced absorption xs.

Reply:
Good point.  The checkvalue() implies that nu must be False, but as I think about it, I can't see why that is necessary. I guess anywhere the transport-xs is calculated, there is a need to allow for scatter multiplication. 

If I allow nu=True, do I have to add a 'nu-diffusion-coefficient' rxn_type?

Also, everywhere where nu is used, the docstrings say "defaults to True". But in all the __init__ functions, I see that nu=False by default. Is that an inconsistency I should fix up everywhere?

Reply:
> If I allow nu=True, do I have to add a 'nu-diffusion-coefficient' rxn_type?
Yeah, I think that would be the way to go.

> Also, everywhere where nu is used, the docstrings say "defaults to True". But in all the init functions, I see that nu=False by default. Is that an inconsistency I should fix up everywhere?
Yeah, but change the docstring itself as opposed to changing the default.